### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by stoffern**
Update dependencies to latest:
`react-native-drawer-layout-polyfill`

Solves: 
https://github.com/react-community/react-navigation/issues/763

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -76,8 +76,8 @@
     "hoist-non-react-statics": "^1.2.0",
     "path-to-regexp": "^1.7.0",
     "prop-types": "^15.5.8",
-    "react-native-drawer-layout-polyfill": "1.2.0",
-    "react-native-tab-view": "^0.0.61"
+    "react-native-drawer-layout-polyfill": "1.3.0",
+    "react-native-tab-view": "^0.0.62"
```
Comment (onlydave) [inline package.json]: 0.0.62 does fix the endless loop issue on iOS but breaks nested tabs again on Android

**Comment by stoffern** [comment]
@onlydave reverted

**Comment by onlydave** [comment]
@satya164 Might be able to shed some light on it. 

I've been poking around for the last hour and it seems to work if you disable animations and swipe on all the tabs. Parent and nested. Or if you set `renderPager = (props: *) => <TabViewPagerPan {...props} />;` in TabView.js. Not familiar enough with react-native-tab-view to know what's going on properly.

**Comment by satya164** [comment]
@onlydave I'm not sure what exactly you mean, but it sounds like this https://github.com/react-native-community/react-native-tab-view/issues/225

In that case, it's a very old issue with viewpager and it's just that you're running into it now (probably something with the react upgrade caused it to resurface).

**Comment by onlydave** [comment]
@satya164 Sorry, should have updated my comment after I finished up. 

All I changed in the end was react-native-tab-view to 0.0.62. To see if it would fix the issue with endless looping when moving between tabs quickly. React and react native are at 16.0.0-alpha.6 and 0.44.0. Didn't change them.

Mayabe more closely related to #1234 

I'll try #1327 and see how that goes

**Comment by satya164** [comment]
@onlydave make sure to open an issue with a gif/video if you repro it

**Comment by stoffern** [comment]
Updated now @satya164, my bad

**Comment by satya164** [comment]
you need to rebase. there are conflicts

**Comment by stoffern** [comment]
@satya164 dont know how you rebase conflics.
Please note that this bug renders the `react-navigation` drawer quite useless on Android

**Comment by skevy** [comment]
Done in https://github.com/react-community/react-navigation/commit/f3a958dca1180af87611a35fb64dbd8dd57da898

Thanks for your work on this.

**Pull request closed without merging**